doc/operators/composite_materials.rst

```diff
@@ -0,0 +1,195 @@
+
+    In this equation, $\mathbf{K}$ is the second-order permeability tensor,
+    $\mu$ is the gas viscosity, $\epsilon$ is the void fraction and $P$ is
+    the gas pressure.
+
+    The
+    :attr:`~mirgecom.fluid.ConservedVars.energy`
+    of the bulk material is given by
+
+    .. math::
+        \frac{\partial \rho_b e_b}{\partial t}
+        + \nabla \cdot (\epsilon_{g} \rho_g h_g \mathbf{u})
+        = \nabla \cdot \left( \bar{\boldsymbol{\kappa}} \nabla T \right)
+        + \mu \epsilon_{g}^2 (\bar{\mathbf{K}}^{-1} \cdot \vec{v} ) \cdot \vec{v}
+        \mbox{ .}
+
+    The first term on the RHS is modeled as an effective diffusive transfer
+    using Fourier's law, where the thermal conductivity is given by
+    $\bar{\boldsymbol{\kappa}}$. The second term on the RHS account for the
+    viscous dissipation in the Darcy's flow. The sub-index $b$ is the bulk
+    material consisting of both solid and gas phases, where
+
+    .. math::
+        \rho_b e_b = \epsilon_{g} \rho_g e_g + \epsilon_s \rho_s e_s \mbox{ .}
+
+    The energy of the gas is given by $e_g(T) = h_g(T) - \frac{P}{\rho_g}$,
+    where $h_g$ is its enthalpy.
+
+    From the conserved variables, it is possible to compute the decomposition
+    status, denoted by
+    :attr:`~mirgecom.wall_model.WallDependentVars.tau`.
+    This yields the proportion of virgin (unpyrolyzed material) to char (fully
+    pyrolyzed) and, consequently, the different thermophysicochemical
+    properties of the solid phase. Thus, the instantaneous material properties
+    depend on the current state of the material, as well as the 
+    :attr:`~mirgecom.eos.GasDependentVars.temperature`.
+    It is evaluated using Newton iteration based on both
+    :attr:`~mirgecom.materials.tacot.GasProperties.enthalpy` (tabulated
+    data) or 
+    :attr:`~mirgecom.eos.PyrometheusMixture.get_internal_energy` (Pyrometheus)
+    and
+    :attr:`~mirgecom.wall_model.WallEOS.enthalpy`,
+    as well as their respective derivatives, namely
+    :attr:`~mirgecom.materials.tacot.GasProperties.heat_capacity` and
+    :attr:`~mirgecom.wall_model.WallEOS.heat_capacity`.
+
+    In *MIRGE-Com*, the solid properties are obtained by fitting polynomials
+    to tabulated data for easy evaluation of the properties based on the
+    temperature. The complete list of properties can be find, for instance, in
+    :mod:`~mirgecom.materials.tacot`.
+    Different materials can be incorporated as separated files.
+    
+    The
+    :class:`~mirgecom.materials.tacot.GasProperties` are obtained based on
+    tabulated data, assuming chemical equilibrium, and evaluated with splines
+    for interpolation of the entries. However, the data is currently obtained
+    by PICA.
+
+.. important ::
+
+    The current implementation follows the description of [Lachaud_2014]_ 
+    for type 2 code. Additional details, extensive formulation and references
+    are provided in https://github.com/illinois-ceesd/phenolics-notes
```

Comment:
Should this go at the top instead of here? In the built docs it ends up towards the bottom, but it seems too important to be down there? Just a thought.